selfhost/utility.jakt

```diff
@@ -279,4 +223,24 @@ function interpret_escapes(anon s: String) throws -> String {
     }
 
     return builder.to_string()
-}
\ No newline at end of file
+}
+
+function null<T>() -> raw T {
```

Comment:
this and allocate are duplicated in posix_process.jakt

Reply:
`allocate` there is calloc, this one is malloc.
will dedup null though.